### PR TITLE
Add project dependency automations

### PR DESCRIPTION
**Pull request by in-op**
This adds the automations checklist to project dependency add and delete, so users can select and execute automations as needed.

### CODE REVIEW

File: src/js/views/Project/Dependencies.jsx [hunk 1]
```diff
@@ -88,12 +66,38 @@ function Dependencies({ project, urlPath }) {
...
+  }
+
   function getDependency(id) {
     return rows.find((row) => row.id === id)?.name
   }
 
   useEffect(() => {
     updateDependencies()
+    getAutomations('create-project-dependency', setCreateAutomations)
+    getAutomations('remove-project-dependency', setRemoveAutomations)
```
Comment (dave-shawley) [inline src/js/views/Project/Dependencies.jsx:100]: You can fetch multiple dependency types at the same time (eg, `category=create-project-dependency&category=remove-project-dependency`) ... _however_ it doesn't look like I return the automation category in the response 🤦

File: src/js/views/Project/Dependencies.jsx [hunk 2]
```diff
@@ -7,6 +7,7 @@ import { httpDelete, httpGet, httpPost, lookupNamespaceByID } from '../../utils'
 import { useTranslation } from 'react-i18next'
 import { ModalForm } from '../../components/Form/ModalForm'
 import { jsonSchema } from '../../schema/ProjectDependencies'
+import { AutomationList } from './Create/AutomationList'
```
Comment (dave-shawley) [inline src/js/views/Project/Dependencies.jsx:10]: Should we move _AutomationList.jsx_ out of _Project/Create_ and into _Project_ at this point?

File: src/js/views/Project/Dependencies.jsx (hunk 1 above)
Comment (in-op) [reply src/js/views/Project/Dependencies.jsx:100]: Yep that's why I did this. I'll see if I can get the endpoint to return it.

File: src/js/views/Project/Dependencies.jsx (hunk 2 above)
Comment (in-op) [reply src/js/views/Project/Dependencies.jsx:10]: Yeah probably a good call.

**Pull request merged**